### DISCUSSION TITLE
Can't get CI/CD working in Octeto.

### DISCUSSION THREAD

**Discussion by 0xrpj** [okteto/okteto#2487] [Q&A]
I have a RASA chatbot hosted in my github. Upon connecting okteto, on the first run, everything runs fine. However, If I try to update something to the github code and redeploy on the app, I get the following error:

<img width="566" alt="image" src="https://user-images.githubusercontent.com/56220537/161378304-2ca90442-39e1-406d-81b3-2d6931af9841.png">

This is my docker compose:
https://github.com/rsnpj/ks-agent/blob/master/docker-compose.yml

This is okteto.yml
https://github.com/rsnpj/ks-agent/blob/master/okteto.yml

**Comment by jLopezbarb** [comment]
Hi,I have not been able to reproduce it from the CLI nor from the UI,  can you check that the problem persists?

**Reply by 0xrpj** [reply]
Yes It does still persist

**Reply by pchico83** [reply]
@rsnpj are you redeploying from the Okteto UI?

**Reply by 0xrpj** [reply]
Yes

**Reply by pchico83** [reply]
@rsnpj the repo isn't public, could you share the relevant parts of you compose/okteto manifest? we cannot repro it
